lib/ansible/modules/source_control/git.py

```diff
@@ -403,10 +442,9 @@ def get_submodule_versions(git_path, module, dest, version='HEAD'):
     return submodules
 
 
-def clone(git_path, module, repo, dest, remote, depth, version, bare,
-          reference, refspec, verify_commit):
+def clone(git_path, module, result, repo, dest, remote, depth, version, bare,
```

Comment:
Don't ever inject new arguments in the middle of signature, always add them in the end. This way you can often break APIs.